x-pack/plugins/security_solution/public/flyout/document_details/shared/hooks/use_graph_preview.ts

```diff
@@ -77,6 +83,7 @@ export const useGraphPreview = ({
     actorIds.length > 0 &&
     eventIds.length > 0 &&
     targetIds.length > 0;
+  const isAlert = Boolean(getFieldsData(ALERT_UUID));
 
-  return { timestamp, eventIds, actorIds, action, targetIds, hasGraphRepresentation };
+  return { timestamp, eventIds, actorIds, action, targetIds, hasGraphRepresentation, isAlert };
```

Comment:
instead of returning `isAlert` here, you could retrieve it directly from within your components using the following:
```typescript
const { dataFormattedForFieldBrowser } = useDocumentDetailsContext();
const { isAlert } = useBasicDataFromDetailsData(
  dataFormattedForFieldBrowser
);
```
This is how we retrieve it in all the other components in the flyout